docker/runit/init.sh

```diff
@@ -32,44 +32,5 @@ sed -i "s,<PROTOCOL>,${SPARK_SSL_PROTOCOL}," /etc/nginx/conf.d/spark.conf
 # bytes cipher strings of Java.
 # sed -i "s,<ENABLED_ALGORITHMS>,${SPARK_SSL_ENABLEDALGORITHMS//,/:}," /etc/nginx/conf.d/spark.conf
 
-# extract cert and key from keystore, write to /etc/nginx/spark.{crt,key}
-if [[ "${SPARK_SSL_ENABLED}" == true ]]; then
-	KEYDIR=`mktemp -d`
-	trap "rm -rf $KEYDIR" EXIT
-
-	echo "${SPARK_SSL_KEYSTOREBASE64}" | base64 -d > "$KEYDIR/spark.jks"
-	ALIAS=$(keytool -list -keystore "$KEYDIR/spark.jks" -storepass "${SPARK_SSL_KEYSTOREPASSWORD}" | grep PrivateKeyEntry | cut -d, -f1 | head -n1)
-	if [[ -z "${ALIAS}" ]]; then
-		echo "Cannot find private key in keystore"
-		exit 1
-	fi
-
-	# convert keystore to p12
-	keytool -importkeystore -srckeystore "$KEYDIR/spark.jks" -srcalias "${ALIAS}" \
-		-srcstorepass "${SPARK_SSL_KEYSTOREPASSWORD}" -destkeystore "$KEYDIR/spark.p12" \
-		-deststorepass "${SPARK_SSL_KEYSTOREPASSWORD}" -deststoretype PKCS12
-
-	# export cert and key from p12
-	openssl pkcs12 -nokeys -passin pass:"${SPARK_SSL_KEYSTOREPASSWORD}" -in "$KEYDIR/spark.p12" -out /etc/nginx/spark.crt
-	openssl pkcs12 -nocerts -nodes -passin pass:"${SPARK_SSL_KEYSTOREPASSWORD}" -in "$KEYDIR/spark.p12" -out /etc/nginx/spark.key
-	chmod 600 /etc/nginx/spark.{crt,key}
-
-	rm -rf "$KEYDIR"
-fi
-
-# Move hadoop config files, as specified by hdfs.config-url, into place.
-if [[ -f hdfs-site.xml && -f core-site.xml ]]; then
-    mkdir -p "${HADOOP_CONF_DIR}"
-    cp hdfs-site.xml "${HADOOP_CONF_DIR}"
-    cp core-site.xml "${HADOOP_CONF_DIR}"
-fi
-
-# Move kerberos config file, as specified by security.kerberos.krb5conf, into place.
-# this only affects the krb5.conf file for the dispatcher
-if [[ -n "${SPARK_MESOS_KRB5_CONF_BASE64}" ]]; then
```

Comment:
This is now all handled in `spark-env.sh` right?

Reply:
Yes